lib/ecto/repo.ex

```diff
@@ -414,12 +414,18 @@ defmodule Ecto.Repo do
 
   ## Options
 
+    * `:prefix` - The prefix to run the query on (such as the schema path
+      in Postgres or the database in MySQL). This will only be applied if no
+      `@schema_prefix` has previously been set.
```

Comment:
I think the more precise definition is: "This will be applied to all from and joins in the query that did not have a prefix explicitly given (either via the :prefix option on join/from or via `@schema_prefix` in the schema)".

Reply:
Ping. :)

Reply:
:flushed: updated!